content/en/docs/advanced-tasks/fail-forward-upgrades.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "Opting into fail forward upgrades"
+linkTitle: "Opting into fail forward upgrades"
+weight: 4
+description: >
+  Fail forward upgrades is a feature that allows for the automatic upgrade of failed installs and upgrades. 
+  It is ultimately unsafe and should only be used in specific circumstances. 
+---
+
+## Warning
+
+Enabling fail forward upgrades may have severe consequences and recovery from these consequences is not guaranteed. Critical versions of operators may not be installed and can thus lead to unknown behavior and potentially data loss. As a result, using this features is not recommended in the majority of circumstances. Only use this feature if:
+
+- You have complete control over operator installs, including OLM.
+- Do not want to manually upgrade your failed operators.
```

Comment:
```suggestion
- You do not want to manually upgrade your failed operators.
```

Reply:
Or, move the "You"s behind the ':'. i.e. `Only use this feature if you:`

Reply:
Ah, good point!